<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opennanoyun</class>
 <widget class="QMainWindow" name="opennanoyun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>opennanoyun</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#00CCFF;
background-color: #669999</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lb_appkey">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Appkey:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_appsecret">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Appscert:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QListView" name="lst_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(228, 119, 255);
background-color: rgb(210, 164, 255);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_downland">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 0);</string>
    </property>
    <property name="text">
     <string>Downland</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_uplaod">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_link">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 139, 244);
background-color: rgb(255, 151, 170);</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_space">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Spacename:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>135</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="le_key">
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 0, 0);
background-color: rgb(20, 127, 111);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_scert">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 142, 119);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_space">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
